idp-core/src/main/java/io/mosip/idp/core/dto/IdPTransaction.java

```diff
@@ -36,6 +36,4 @@ public class IdPTransaction implements Serializable {
 
     String nonce;
     String state;
-
-    String individualId; //TODO need to encrypt (required for IDA kyc-exchange call)
```

Comment:
As discussed, lets keep this configurable to be added and encrypted. Lets not move it to the wrapper library because we want to hit to redis again there.

Reply:
Done

Reply:
reverted the change